pkg/controller/plan/adapter/ovirt/builder.go

```diff
@@ -391,7 +420,23 @@ func (r *Builder) mapCPU(vm *model.Workload, object *cnv.VirtualMachineSpec) {
 	if vm.CpuPinningPolicy == model.Dedicated {
 		object.Template.Spec.Domain.CPU.DedicatedCPUPlacement = true
 	}
+	if vm.CustomCpuModel != "" {
+		r.setCpuFlags(vm.CustomCpuModel, object)
+	}
+}
 
+func (r *Builder) setCpuFlags(fullCpu string, object *cnv.VirtualMachineSpec) {
+	object.Template.Spec.Domain.CPU.Model = cpuMap[fullCpu]
```

Comment:
the idea here is to use the same CPU as the VM would have in oVirt, right? so why not [to take the first item as the model ](https://github.com/oVirt/ovirt-engine/blob/master/backend/manager/modules/vdsbroker/src/main/java/org/ovirt/engine/core/vdsbroker/builder/vminfo/LibvirtVmXmlBuilder.java#L452)?